vault/config/init.sls

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls syntax=yaml softtabstop=2 tabstop=2 shiftwidth=2 expandtab autoindent
+
+{% from "vault/map.jinja" import vault with context -%}
+
+vault-config-init-file-serialize:
+  file.serialize:
+    - name: /etc/vault/conf.d/config.json
+    - encoding: utf-8
+    - formatter: json
+    - dataset: {{ vault.config | json }}
+    - user: root
+    - group: vault
+    - mode: 640
+    - makedirs: True
+    - watch_in:
+      - service: vault
```

Comment:
Refer to [this comment](https://github.com/saltstack-formulas/vault-formula/pull/23/files?file-filters%5B%5D=.sls&file-filters%5B%5D=.yaml&hide-deleted-files=true#r268416313).